src/browser/index.jsx

```diff
@@ -395,6 +401,12 @@ var MattermostView = React.createClass({
       }
     });
 
+    webview.addEventListener('page-title-updated', function(event) {
```

Comment:
Well I could just filter out any title starting with ```* ``` but that will fail if anybody has a channel that starts like that. (hopefully unlikely)